packages/graphql-cli-graphback-plugin/src/commands/backend-init.ts

```diff
@@ -0,0 +1,28 @@
+import yargs from 'yargs';
+import { init } from 'graphback-cli'
+
+type Params = { name?: string, templateName?: string, templateUrl: string }
+
+export const command = 'backend init <name>'
```

Comment:
cc @wtrocki @ankitjena we prefixed the commands with `backend` so we wouldn't clash with the other commands provided by `graphql-cli`. What do you think of these ones? I would prefer if we could figure out how to shorten those names but still keep them meaningful. Any ideas?

Reply:
hmm.. requirements are - single plugin single command - so whatever we will use - backend or anything else as prefix will work.